packages/manager/modules/log-to-customer/src/live-tail/routing.js

```diff
@@ -0,0 +1,25 @@
+import { SERVICES } from '../constants';
+
+export default /* @ngInject */ ($stateProvider) => {
+  $stateProvider.state('pci.projects.project.kubernetes.details.logs', {
+    url: '/logs',
+    views: {
+      'kubernetesView@pci.projects.project.kubernetes.details': 'liveTail',
+    },
+    params: {
+      logApiUrl: null,
+      logKindApiUrl: null,
+    },
+    resolve: {
+      breadcrumb: /* @ngInject */ ($translate) =>
+        $translate.instant('log_tab_title'),
+      service: /* @ngInject */ ($transition$) => {
+        return (
+          SERVICES.find((service) =>
+            $transition$.to().name.includes(service),
+          ) || 'kubernetes'
```

Comment:
```suggestion
         service || 'kubernetes'
```

Reply:
I can remove SERVICES since there is only one value for now, but I'd prefer to keep the $transitions like this maybe :
```suggestion
         $transition$.to().name.includes(service) || 'kubernetes'
```
So the way to detect service from state is in place

Reply:
See next comment
-> For now there is only one service, but I'd prefer to keep if that way so the architecture and the way to retrieve service is already in place and known